CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v2.6.0 (unreleased)
+### Notes
+- This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20).
+
+### Oneview Features supported
+- Encloure
```

Comment:
Please correct the spelling to Enclosure

Reply:
done!